stylesheets/_component.forms.scss

```diff
@@ -6,6 +6,32 @@ fieldset {
 
 .form {
     background-color: color(white);
+
+    // Added by @stanton, not used in CXM - safe to remove?
+    .slide-inner & {
+        padding-top: 17px;
```

Comment:
I'm almost certain `.slide-inner` is a class from the old decks component, I can't find it in use in CMS or Pulsar so should be ok to remove